add spring boot troubleshooting guide (#7058)
open-telemetry/opentelemetry.io: content/en/docs/zero-code/java/spring-boot-starter/troubleshooting.md

---
title: Troubleshooting
weight: 80
---

## ConverterNotFoundException

Error:

```text
org.springframework.core.convert.ConverterNotFoundException: No converter found capable of converting from type [java.lang.String] to type [java.util.Map<java.lang.String, java.lang.String>]
```

> This is caused ty application has replaced the default converter registry as
> follows - or similar:

Reviewer: ```suggestion
This error is caused when the application replaces the default converter registry. For example:
```

```java
@Bean
public ConversionService conversionService() {
    return new DefaultFormattingConversionService();
}
```

> To fix the issue, change it to:

Reviewer: ```suggestion
To fix the issue, change the converter registry to:
```
